poker-pals/src/components/Tables.vue

```diff
@@ -1,42 +1,90 @@
 <template>
-    <div id="tables-page">
-        <UserNavbar class="navbar-section"/>
-        <div class="table">
-            <div class="table-description">
-                <div class = "table-stakes">
-                    <p> 20,000 </p> 
-                    <p> 1000/2000 </p>
-                </div> 
-                <div class = "chips-pic">
-                <img alt="Vue logo" src="../assets/chip.png">
-                </div> 
+<!-- Table list div --> 
+  <div class = "table-list">
+    <UserNavbar class="navbar-section"/>
+    <ul>
+    <!-- Loop through tables and display --> 
+    <li v-for="(tableIndex, tableNum) in tables" :key="tableIndex.id">
+      <!-- Join room on click --> 
+      <div class="table"  id = "table-click" v-on:click="joinRoom(table.id, tableNum)">
+        <!-- Display table's buyin and blinds --> 
+        <div class="table-description">
+            <div class = "table-stakes">
+                <p>{{tableIndex.table.buyIn}}</p> 
+                <p>{{tableIndex.table.bigBlind}}</p>
             </div> 
-            <div class="table-details">
-                <div class = "table-name">
-                    <p> Table 1</p>
-                </div>
-                <div class = "seats">
-                    <p> 6/6 </p>
-                </div>
+            <div class = "chips-pic">
+              <img alt="Chip picture" src="../images/chip.png">
+            </div> 
+        </div> 
+        <!-- Display table's name and seat availability --> 
+        <div class="table-details">
+            <div class = "table-name">
+                <p> {{tableIndex.table.tableName}}</p>
+            </div>
+            <div class = "seats" id = "seatsId">
+            <!-- TODO: Update seat availability as seats fill up.--> 
+                <p> {{tableIndex.table.numberOfTableSeats}}/{{tableIndex.table.numberOfTableSeats}}</p>
             </div>
         </div>
-    </div>
+      </div>
+    </li>
+    </ul>
+  </div>
 </template>
 
 <script>
+import io from "socket.io-client";
 import UserNavbar from "./navbars/UserNavbar";
 
 export default {
   name: 'Tables',
   components: {
       UserNavbar
+  },
+  created() {
+        this.socket = io("http://localhost:3000");
+  },
+
+  data() {
+    return {
+      socket: {},
+      tables: [],
+    };
+  },
+
+  mounted() {
+    //Retrieve the current tables available and store into tables array 
+      this.socket.on("serve-tables", rooms => {
+          this.tables = rooms; 
+          console.log('These are the tables');
+          console.log(this.tables);
```

Comment:
Would be good to comment out and use for debugging only. Or you could also make a global DEBUG variable and guard these logs with that variable. If you choose the latter, let me know if you need help with it since I've already researched quite a bit on it.

Reply:
this code is from Freds pr, it temporary got added to solve the merge conflict, its removed now